M/MAGEMin/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MAGEMin"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ComputationalThermodynamics/MAGEMin.git", "032116744f3df05b5a2999c2eb39689c831cf4c7"),
+    ArchiveSource("https://github.com/Reference-LAPACK/lapack/archive/refs/tags/v3.10.0.tar.gz", "328c1bea493a32cac5257d84157dc686cc3ab0b004e2bea22044e0a59f6f8a19"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/lapack-*/LAPACKE/
+cmake ../ -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DLAPACKE=ON -DBUILD_SHARED_LIBS=ON
+make lapacke -j${nproc}
+make install
+
+cd ../../MAGEMin/
+if [[ "${target}" == *-mingw* ]]; then
+    MPI_LIBS="-lmsmpi"
+else
+    MPI_LIBS="-lmpi"
+fi
+make CC=$CC CCFLAGS="-O3 -g -fPIC -std=c99" LIBS="-L${libdir} -lm -llapacke -lnlopt ${MPI_LIBS}" INC="-I${includedir}" lib
+install -Dvm 755 libMAGEMin.dylib "${libdir}/libMAGEMin.${dlext}"
+install -Dvm 644 src/*.h "${includedir}"
+install_license LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+platforms = expand_gfortran_versions(platforms)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libMAGEMin", :MAGEMin_library)
```

Comment:
Can we call this
```suggestion
    LibraryProduct("libMAGEMin", :libMAGEMin)
```
?

Reply:
Sure